config.yml

```diff
@@ -41,6 +40,7 @@ debug: false
 # are deafened (i.e. puppets do not need to hear anything!)
 irc_puppet_prejoin_commands:
   - "MODE ${NICK} +D" # Note that for inspircd 3.x this should be +d!
+# - "PRIVMSG NickServ IDENTIFY ${NICK} 123456" # identifying to nickserv should be handled here!
```

Comment:
```suggestion
# - "PRIVMSG NickServ IDENTIFY your-password-here" # this is how you can identify to NickServ!
```

Reply:
I included ${NICK} because although most people just do "/ns identify pass" if you ping out then reconnect and become `$you + '_'` then "/ns identify pass" fails whereas "/ns identify ${NICK} pass" would work in such a case. Regardless, people can figure that part out easily themselves.